blockstore.go

```diff
@@ -34,7 +34,7 @@ func (e *exchangeBsWrapper) GetBlock(ctx context.Context, c cid.Cid) (blocks.Blo
 	defer cancel()
 
 	if goLog.Level().Enabled(zapcore.DebugLevel) {
-		goLog.Debugw("block requested from saturn", "cid", c.String())
+		goLog.Debugw("block requested from strn", "cid", c.String())
```

Comment:
:information_source: consistent with caboose errors

Reply:
(should we spell it out?)

Reply:
I mean, someone else made this decision: https://strn.pl/ipfs/bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi :upside_down_face: